components/sftp/watch-remote-dir-for-new-file.js

```diff
@@ -0,0 +1,33 @@
+const sftpPipedreamApp = require('https://github.com/skilbjo/pipedream/components/sftp/sftp.app.js'); // TODO s/skilbjo/PipedreamHQ/
+
+module.exports = {
+  name: "watch a remote directory for a new file",
```

Comment:
We need to better document this convention, but I'd shorten the name to something that distinguishes the event emitted, e.g. "New File in Remote Directory".

Then you can add a `description` property that elaborates. Normally we start the description with "Emits [event X]" and follow with any source-specific details instructions. [Spotify](https://github.com/PipedreamHQ/pipedream/blob/master/components/spotify/new-saved-tracks.js#L4) has a good example.